nixos/modules/misc/ids.nix

```diff
@@ -284,6 +284,7 @@
       glance = 266;
       couchpotato = 267;
       gogs = 268;
+      trezord = 269;
```

Comment:
But the trezor is a hardware device, correct? Static uids are for ensuring consistent ownership across NixOS deployments.